geometry/TrianglesView.swift

```diff
@@ -8,12 +8,14 @@ import UIKit
 
 class TrianglesView: UIView {
     private var dataSource: TrianglesProtocol?
-
+    var redt: CGFloat = 0.0
```

Comment:
The letter "t" as abbreviation is unknown to anybody but the author of this code. Only well known abbreviations related to business area or programming in general are acceptable.